lib/connect-backbone-to-react.js

```diff
@@ -128,6 +128,18 @@ module.exports = function connectBackboneToReact(
             (this.context.models && this.context.models[mapKey] === this.models[mapKey])) {
             return; // Did not change.
           }
+          // Do not attempt to create event listeners on an undefined model.
+          if (!model) {
+            // Instead, if it was previously defined, remove the old listeners.
+            const prevModel = this.props.models[mapKey];
+            if (prevModel) {
+              getEventNames(mapKey).forEach(name => {
```

Comment:
This is duped now, let's create a util method for this functionality. `removeListeners(mapKey, model) {}`.